Frontend/src/app/views/auth/login/login.component.ts

```diff
@@ -78,9 +78,11 @@ export class LoginComponent implements OnInit {
               if(response.message == "Added User to Project"){
                 this.toastrService.success("You have been added to a project", "Success");
               }else if(response.message == "Project not found"){
-                this.toastrService.danger("Invite code has not been found", "Error")
+                this.toastrService.danger("Invite code is not valid", "Error")
               }else if (response.message == "Unable to add to Archived Project"){
                 this.toastrService.danger("Unable to add user to Archived Project", "Error");
+              }else if (response.message == "User already in project"){
+                this.toastrService.danger("User is already assigned to this project", "Error");
```

Comment:
I feel like this wording is a bit off, we are talking directly to the user this seems more of like a backend logging message.